.github/CODEOWNERS

```diff
@@ -81,4 +81,5 @@ mypy.ini                            @DataDog/python-guild  @DataDog/apm-core-pyt
 .github/PULL_REQUEST_TEMPLATE.md    @DataDog/python-guild  @DataDog/apm-core-python
 .github/ISSUE_TEMPLATE.md           @DataDog/python-guild  @DataDog/apm-core-python
 .github/CODEOWNERS                  @DataDog/python-guild  @DataDog/apm-core-python
+.github/workflows/system-tests.yml  @DataDog/python-guild  @DataDog/apm-core-python
```

Comment:
Does this backport work for 1.18? Seeing that the CODEOWNERS file contains errors. Or is that just a red herring?